modules/src/cpcontest.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import os
+from templates.text import TextTemplate
+
+
+def process(input, entities):
+    output = {}
+    try:
+
+        r = requests.get('https://cpcontest-api.herokuapp.com/')
```

Comment:
Is this your personal api? If yes, do you plan to maintain it (hosting, bug fixes, etc.)?